pkr/kard.py

```diff
@@ -103,33 +93,62 @@ def list(cls, kubernetes=False):
         return kards
 
     @classmethod
-    def create(cls, kard_name, env_name, driver_name, extra):
+    def create(cls, name, env, driver, features, meta, extra, **kwargs):
         """Factory method to create a new kard"""
+        extras = {"features": []}
+        if meta:
+            extras.update(yaml.safe_load(meta))
+        extras.update({a[0]: a[1] for a in [a.split("=", 1) for a in extra]})
+        for feature in features_merge(extras["features"]):
```

Comment:
features_merge function name is not explicit. You should rename `features_merge`.

Reply:
Same name right ?

Reply:
Handled

Reply:
Great !